oresat_dxwifi/services/camera.py

```diff
@@ -0,0 +1,95 @@
+"""Oresat Live Camera Service"""
+
+from olaf import Service, logger
+from enum import IntEnum
+from ..camera.camera import CameraInterface
+
+
+class State(IntEnum):
+    OFF = 0
+    BOOT = 1
+    STANDBY = 2
+    FILMING = 3
+    ERROR = 0xFF
+
+
+STATE_TRANSISTIONS = {
+    State.OFF: [State.BOOT],
```

Comment:
Is a transition back to `OFF` needed (perhaps from `ERROR`)? Not entirely sure how the state machine works